src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
We might be able to specify other ways to represent the
multiple-parent case, or specify it away.  As soon as we do a
pull request, I am going to get Lincoln to look at it.

I am ambivalent on the parent vs child specification, but would
like to document the rational for a choice.  Please explain your
preference so we can document it.

From my experience, top to bottom is a common traversal, but you
need to be able to do both.  Although either can be handled with
queries.

I also like the conceptual model of being able to combine
immutable nodes into different trees, which parent doesn't
allow.

I am inclined to put in both in the pull request and sort it out
in a larger audience.



Uri Laserson <notifications@github.com> writes:
> In src/main/resources/avro/sequenceAnnotations.avdl:
> 
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
> 
> +1 to parent over child.
> 
> Is there a need to allow an array of parents?
> 
> —
> Reply to this email directly or view it on GitHub.*
>